src/Makefile.am

```diff
@@ -1,3 +1,3 @@
 ## Process this file with automake to produce Makefile.in
 include $(top_srcdir)/config/Make-rules
-SUBDIRS = IB fortran adv_diff advect level_set navier_stokes utilities . tools
+SUBDIRS = IB fortran adv_diff advect level_set navier_stokes utilities . tools wave_tank
```

Comment:
same here, though that was preexisting.

Reply:
Yeah, I think either it was either me or Nishant. But we followed the existing Makefile.am pattern, IIRC.